src/main/webapp/script.js

```diff
@@ -47,21 +47,27 @@ function cleanseUrl(url) {
 /**
  * Fetches data and adds to html
  */
-function getChart() {
-  $('form').submit(function() {
+async function getChart() {
+  $('form').submit(async function() {
+    commentStats = await getYouTubeComments();
+    averageScore = commentStats.averageScore;
+    aggregatedValues = commentStats.aggregateValues; 
+
     const CommentSentimentTable = new google.visualization.DataTable();
     CommentSentimentTable.addColumn('string', 'SentimentRange');
     CommentSentimentTable.addColumn('number', 'CommentCount');
 
-    for (currentLabel = LOWEST_SCORE; currentLabel < HIGHEST_SCORE; currentLabel += INTERVAL) {
-      // TODO: Replace abritrary value 6 with correct aggregation value  
+    currentLabel = LOWEST_SCORE;
+    // The json keys (ranges of scores) are sorted through their starting values
+    // using subtraction comparison  
+    Object.keys(aggregatedValues).sort((a, b) => a - b).forEach(function(key) {
       CommentSentimentTable.addRows([
-          [(Math.round(currentLabel * 10) / 10).toString(), null],
-          [null, Math.random()*10]
+          [key, null],
```

Comment:
Before this change, this `sentimentRange` column was populated w/ the lower bound of the "range" and now it's populated with a JSON object.

Can we change this to contain a String representation of the Range? I.e. `0.0 - 0.2`

Maybe we'll need to define a new method like `toRangeStringRepresentation` that will convert Range JSON objects to a String representation that we will want to display to the user.